docs/design/jobflow/README.md

```diff
@@ -383,6 +387,5 @@ https://www.bilibili.com/video/BV1c44y1Y7FX
 * `if` statements
 * `switch` statements
 * `for` statements
-* Support job failure retry in JobFlow
```

Comment:
We should also implement this in controller first?

Reply:
The next PR to add this feature.

This PR to  patch  MaxRetry from jobflow to jobTemplate